### PR TITLE
next round of paper edits

### PR DESCRIPTION
**Pull request by paxtonfitzpatrick**
includes small changes to cover letter, and edits & fixes to paper and figs up through line 315 (in my version of the compiled PDF, where example7.pdf appears)

### CODE REVIEW

File: paper/main.tex [hunk 1]
```diff
@@ -679,20 +677,19 @@ \section{Illustrative Example}\label{sec:illustrative-example}
...
-v0.21.3).  In that version of \texttt{scikit-learn}, the
-\texttt{LatentDirichletAllocation} class is defined in
-\texttt{sklearn.decomposition.\_lda}.  However, in version
+v0.21.3.  In that version of \texttt{scikit-learn}, the
+\texttt{LatentDirichletAllocation} class was defined in
+\texttt{sklearn.decomposition.online\_lda}.  However, in version
 0.22.0, that module was renamed to \texttt{\_online\_lda}, and in
-versions 0.22.1 and higher, the name was reverted back to
-\texttt{\_lda}.
+version 0.22.1, it was again renamed to \texttt{\_lda}.
```
Comment (jeremymanning) [inline paper/main.tex:684]: What was the difference between v0.21.3 and 0.22.0?

Seems like .online_lda --> .online_lda --> ._lda.  Maybe a typo?

**Review by jeremymanning — CHANGES_REQUESTED** [review]
these changes look awesome!  nice catches.  i made a couple of formatting/syntax edits, but aside` from that i like all of the changes.  the one (very minor) thing left that i see is in how you modified the history of the `LatentDirichletAllocation` module in `scikit-learn`.  i added a comment in my review to mark the relevant text.

also, i haven't recompiled anything.  i'll leave the pull request open so that you can address that one last comment and recompile the cover letter + main document.  then i think this is good to go from my end!

File: paper/main.tex (hunk 1 above)
Comment (paxtonfitzpatrick) [reply paper/main.tex:684]: it goes `online_lda` --> `_online_lda` (with leading underscore) --> `_lda`

Comment (jeremymanning) [reply paper/main.tex:684]: got it :+1:

**Pull request merged**